### PR TITLE
Small fixes on the last merged pt-br.md PR

### PR DESCRIPTION
**Pull request by Kuron-kun**
Moved a lot of folders to their updated locations and fixed a lot of issues regarding outdated links and outdated info.

### CODE REVIEW

File: wiki/Beatmap_Editor_Guides/Adding_Custom_Hit_Sounds/pt-br.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Você então decide se quer que seja o normal hit, clap, whistle ou finish.
 
 ### 1ª exemplo
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Adding_Custom_Hit_Sounds/pt-br.md]: is this superscript really needed?

File: wiki/Beatmap_Editor_Guides/Adding_Custom_Hit_Sounds/pt-br.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ Se você quiser mais variedade e precisa de mais opções para aplausos e tudo m
 
 ### 2ª exemplo
```
Comment (MegaApplePi) [inline wiki/Beatmap_Editor_Guides/Adding_Custom_Hit_Sounds/pt-br.md]: ditto
> is this superscript really needed?

File: wiki/Achievements/pt-br.md [hunk 3]
```diff
@@ -56,10 +56,10 @@ Skill
 
 As conquistas **Skill** atualmente estão divididos em três grupos: **Combo**, **High Ranker** e **Star-based**
 
-As conquistas **Combo** podem ser obtidas ao alcançar um [Combo](Combo "wikilink") específico no osu! padrão e pode ser obtida com qualquer música.
+As conquistas **Combo** podem ser obtidas ao alcançar um [Combo](wiki/Glossary) específico no osu! padrão e pode ser obtida com qualquer música.
```
Comment (TPGPL) [inline wiki/Achievements/pt-br.md]: `/wiki/Glossary/` missed `/`

File: wiki/Mascots/pt-br.md [hunk 4]
```diff
@@ -231,6 +231,4 @@ Don é o protagonista da série Taiko no Tatsujin. Ele é um tambor de taiko com
 Galeria de imagens
```
Comment (TPGPL) [inline wiki/Mascots/pt-br.md]: wrong header

File: wiki/Mascots/pt-br.md [hunk 5]
```diff
@@ -231,6 +231,4 @@ Don é o protagonista da série Taiko no Tatsujin. Ele é um tambor de taiko com
...
 =============
 
-[Clique aqui para ir à galeria de imagens.](Gallery/en.md "wikilink")
-
-[Category:History of osu!/pt-br](Category:History_of_osu!/pt-br "wikilink")
+[Clique aqui para ir à galeria de imagens.](/wiki/Mascots/Gallery "wikilink")
```
Comment (TPGPL) [inline wiki/Mascots/pt-br.md]: remove wikilink

File: wiki/Registration/pt-br.md [hunk 6]
```diff
@@ -1,43 +1,39 @@
...
-| **1. Nome de usuário**                                                                                                                                       |
-| a. Máximo de 15 caracteres                                                                                                                           |
-| b. Use apenas os caracteres alfanuméricos ("a-z, A-Z, 0-9"), subtraços ("\_"), colchetes, ("\[" e "\]") e traços ("-"). |
-| **2. Endereço de e-mail**                                                                                                                                 |
-| **3. Senha**                                                                                                                                       |
-| **4. Aviso**                                                                                                                                           |
-| **5. Botão de continuar**                                                                                                                                |
+1. Na [página principal](https://osu.ppy.sh), Clique em ``Sou novo!`` para iniciar o processo de registro
```
Comment (TPGPL) [inline wiki/Registration/pt-br.md]: empty line between text and image
```
Text

Image

Text
```

File: wiki/Storyboard_Scripting/Audio/pt-br.md [hunk 7]
```diff
@@ -1,3 +1,6 @@
+Áudio de Storyboard
+=================
```
Comment (TPGPL) [inline wiki/Storyboard_Scripting/Audio/pt-br.md]: make it longer /shrug

File: wiki/Storyboard_Scripting/Variables/pt-br.md [hunk 8]
```diff
@@ -1,34 +1,47 @@
+Variáveis de Storyboard
+====================
```
Comment (TPGPL) [inline wiki/Storyboard_Scripting/Variables/pt-br.md]: make it longer? /shrug

File: wiki/Storyboards/Maps/pt-br.md [hunk 9]
```diff
@@ -121,7 +124,7 @@ Beatmaps que não têm uma alma™, mas que ainda sim são bons
 -   [DJ Sharpnel - StrangeProgram](http://osu.ppy.sh/s/5774)
 -   [Shiori Mikami - Watashi, Shuyaku no Akaza Akari Desu (Short Ver.)](http://osu.ppy.sh/s/48072)
 
-Não rankeados
+Não ranqueados
 ----------
```
Comment (TPGPL) [inline wiki/Storyboards/Maps/pt-br.md]: same here

File: wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/pt-br.md [hunk 10]
```diff
@@ -1,117 +1,117 @@
-Este artigo lista os países que participaram nos vários torneios de osu!, Taiko, CtB e osu!mania, além de sua classificação caso tenham terminado entre as 4 melhores equipes
+Este artigo lista os países que participaram dos vários torneios de osu!, Taiko, CtB e osu!mania que aconteceram no osu!, assim como suas colocações se conseguiram chegar no top 4.
```
Comment (TPGPL) [inline wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/pt-br.md]: headers broken in the whole article

Comment (TPGPL) [inline wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/pt-br.md]: Icons broken in the whole article

Comment (TPGPL) [inline wiki/Tournaments/Countries_that_participated_in_osu!_tournaments/pt-br.md]: Links broken in the whole article /shrug

File: wiki/Mascots/pt-br.md (hunk 4 above)
Comment (Kuron-kun) [inline wiki/Mascots/pt-br.md]: should I remove the header code then?

File: wiki/Registration/pt-br.md (hunk 6 above)
Comment (Kuron-kun) [inline wiki/Registration/pt-br.md]: adding a space actually breaks the code there, better keep it like this since en.md doesn't have spaces as well

**Pull request merged**